setup.py

```diff
@@ -24,6 +24,8 @@
 __url__ = 'https://github.com/idlesign/deluge-webapi'
 
 
+__pkg_data__ = {__plugin_name__.lower(): ["data/*"]}
```

Comment:
Style: here and elsewhere please use consistent string literals quoting.

What I use: 
```
'string literal'
'''multiline string literal'''
"""docstrings"""
```